src/app-nav-bar/logo.js

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+import {StyledLink} from '../link/index.js';
+import {StyledAppName} from './styled-components.js';
+
+export default function Logo(props: {
+  appDisplayName: ?React.Node,
+  link: ?string,
+}) {
+  return (
+    <StyledLink $style={{textDecoration: 'none'}} href={props.link || ''}>
```

Comment:
Accepting an href with no overrides will be problematic when there will inevitably be a request to make this work with react router or similar. My first thought is to reformat this to accept either a string or abstract component in top-level props. If a string, render this else render the provided component? Downside being we may see some non-standard display names in the top-left

Reply:
In this current PR there are not many overrides allowed, but it will be coming in as follow-up changes due to the size of this initial PR.